### ISSUE TITLE
As a SO/CE I would like more control of the positioning of Background Images

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1203]
## Context

Background images are used on hero units and block sections and are automatically positioned in the center of the element.  Most of the time this works, but as the element is viewed at different browser widths and different devices, the presentation of the image could be improved.

## Possible Fix

By allowing the SO/CE to select where the background should be aligned to, we could make it easier to design these elements.

Here's a codepen that demonstrates the idea: https://codepen.io/kevincrafts/pen/JJmzqg

Most often, the top/center/bottom is the most useful, but if a hero unit is compressed enough that it becomes taller than it's width, the right/left can be useful.

**Comment by alexfinnarn** [comment]
PR for resizing window on Behat tests - https://github.com/CuBoulder/express/pull/752
